unitary/quantum_chess/move.py

```diff
@@ -185,9 +185,9 @@ def to_string(self, include_type=False) -> str:
           include_type: also include the move type/variant in the string
         """
         movestr = self.source + self.target
-        if self.is_split_move():
+        if self.is_split_move() and self.target2:
```

Comment:
This change seems suboptimal since if you had a wrongly constructed split move, then the evidence would be covered up when printing it as a string. I think you could do `str(self.target2)` instead

Reply:
Good idea.  Done.